modules/popup-stack/package.json

```diff
@@ -6,7 +6,7 @@
   "license": "Apache-2.0",
   "main": "dist/commonjs/index.js",
   "module": "dist/es6/index.js",
-  "sideEffects": false,
+  "sideEffects": true,
```

Comment:
This should have always been `true`. Since we manipulate `window` outside of a function call, this is required to support tree shaking. Proper tree shaking would break previous versions of `PopupStack`. I'm surprised we haven't received any bug reports.